beacon-chain/blockchain/service.go

```diff
@@ -127,25 +109,20 @@ func (c *ChainService) IncomingBlockFeed() *event.Feed {
 
 // HasStoredState checks if there is any Crystallized/Active State or blocks(not implemented) are
 // persisted to the db.
+// TODO: Remove - only used in tests
 func (c *ChainService) HasStoredState() (bool, error) {
 
-	hasActive, err := c.beaconDB.Has([]byte(activeStateLookupKey))
+	hasCrystallized, err := c.beaconDB.Has([]byte(CrystallizedStateLookupKey))
```

Comment:
Make sure you remove all conversions to `[]byte` since these keys are already bytes now.